Sources/WKData/Models/WKProject.swift

```diff
@@ -1,6 +1,21 @@
 import Foundation
 
-public enum WKProject: Equatable {
+public enum WKProject: Equatable, Hashable, Identifiable, Codable {
+    public var id: String {
+        switch self {
+        case .commons:
+            return "commons"
+        case .wikidata:
+            return "wikidata"
+        case .wikipedia(let language):
+            if let languageVariantCode = language.languageVariantCode {
+                return "wikipedia-" + language.languageCode + "-" + languageVariantCode
+            } else {
+                return "wikipedia-" + language.languageCode
+            }
+        }
+    }
+    
     public static func == (lhs: WKProject, rhs: WKProject) -> Bool {
```

Comment:
Am I missing a reason we can't simplify this conformance to just `return lhs.id == rhs.id`?

Reply:
Nope, this is updated! I think the Equatable conformance came before Identifiable, and I didn't go back and simplify. Good catch.